regression/cbmc/integer-assignments1/test.desc

```diff
@@ -3,5 +3,5 @@ main.c
 --trace --smt2
 ^EXIT=10$
 ^SIGNAL=0$
-^  b=100
+^  b=100 (0b1100100)$
```

Comment:
The parentheses will need escaping. (CI currently passes as we don't have SMT in there yet.)

Reply:
done